streaming/src/test/scala/org/apache/spark/streaming/ReceivedBlockTrackerSuite.scala

```diff
@@ -82,11 +82,6 @@ class ReceivedBlockTrackerSuite
     receivedBlockTracker.allocateBlocksToBatch(2)
     receivedBlockTracker.getBlocksOfBatchAndStream(2, streamId) shouldBe empty
 
-    // Verify that batch 2 cannot be allocated again
```

Comment:
Please add test to verify that `allocateBlocksToBatch(x)` where x<=2 does nothing.